modules/spark/src/main/scala/notebook/spark/util/SparkClosureUtil.scala

```diff
@@ -0,0 +1,9 @@
+package notebook.spark.util
+
+import scala.reflect.runtime.universe
+import org.apache.log4j.AppenderSkeleton
+import org.apache.log4j.spi.LoggingEvent
+
+class SparkClosureUtil(appendTo: String => Unit = println) 
```

Comment:
it's not straightforward what the code does and certainly requires a docstring. maybe:
```scala
/**
*   A closure cleaning util which allows accessing the log output by ` Spark's ClosureCleaner`.
*   It Registers the given `appendTo` fn to be called by Spark's ClosureCleaner. The default is to `println` to stdout.
**/
class SparkClosureUtil(appendTo: String => Unit = println) 

```